config/envs/default.py

```diff
@@ -35,15 +36,4 @@ class DefaultConfig(object):
     if RSA256_PUBLIC_KEY_BASE64:
         RSA256_PUBLIC_KEY = base64.b64decode(RSA256_PUBLIC_KEY_BASE64).decode()
 
-    # TODO: replace with envar
-    ADDITIONAL_EMAIL_LOOKUPS = {
-        # devs can submit for these LAs and places
-        "version1.com": (
-            ("Sunderland City Council", "Worcester City Council"),
-            ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
-        ),
-        "levellingup.gov.uk": (
-            ("Sunderland City Council", "Worcester City Council"),
-            ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
-        ),
-    }
+    ADDITIONAL_EMAIL_LOOKUPS = ast.literal_eval(os.getenv("ADDITIONAL_EMAIL_LOOKUPS", "{}"))
```

Comment:
Do we want to assert that the value is a dictionary? I'd have thought that it will then give us a better error to debug than if we try to update a dictionary with a string on app init

Reply:
added in c10a886